### PR TITLE
Add latest updates

### PR DESCRIPTION
**Pull request by aperacaula**

### CODE REVIEW

File: staff/aperacaula/calculator-app/scripts/logic.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class Calculator {
```
Comment (agandia9) [inline staff/aperacaula/calculator-app/scripts/logic.js:2]: ....?

File: staff/aperacaula/js/challenges/scripts/box.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+debugger
```
Comment (agandia9) [inline staff/aperacaula/js/challenges/scripts/box.test.js]: delete **debugger** before commit...

File: staff/aperacaula/js/challenges/scripts/forEach.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// function forEach(array, action) {
```
Comment (agandia9) [inline staff/aperacaula/js/challenges/scripts/forEach.js:1]: use strict

File: staff/aperacaula/calculator-app/scripts/logic.js (hunk 1 above)
Comment (aperacaula) [reply staff/aperacaula/calculator-app/scripts/logic.js:2]: no entiendo... se me ha perdido todo mi calculator!!

File: staff/aperacaula/rock-paper-scissors-app/scripts/logic.spec.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        expect(game.winner()).toBe('romeo')
+
+        var state = game.state()
+        expect(state.length).toBe(3)
+        expect(state[0]).toEqual({ player1: 'rock', player2: 'paper'})
+        expect(state[1]).toEqual({ player1: 'scissors', player2: 'paper'})
+        expect(state[2]).toEqual({ player1: 'rock', player2: 'scissors'})
+     })
+
+    // it('should player2 win in 6 rounds, after 3 ties', function () {
```
Comment (manuelbarzi) [inline staff/aperacaula/rock-paper-scissors-app/scripts/logic.spec.js:79]: HEY ) why is this test commented? tests should not be touched!

Comment (manuelbarzi) [reply staff/aperacaula/rock-paper-scissors-app/scripts/logic.spec.js:79]: UNCOMMENT these tests!

File: staff/aperacaula/leanyoureact/src/components/App.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  _handlerWriteName= (e) => {
+    this.setState({
+      username: e.target.value
+    })
+  }
+
+  _handlerSearchUser= (e) => {
+   e.preventDefault()
+   console.log('yew, you submitted user')
+   console.log(this.username)
```
Comment (agandia9) [inline staff/aperacaula/leanyoureact/src/components/App.js:38]: clean console.logs!

File: staff/aperacaula/leanyoureact/src/components/searchForm/SearchForm.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function SearchForm(props) {
+  return (
+    <form onSubmit={props._handlerSearchUser}>
```
Comment (agandia9) [inline staff/aperacaula/leanyoureact/src/components/searchForm/SearchForm.js:5]: destructuring props!

**Comment by manuelbarzi** [comment]
please, review comments and take action.

**Pull request merged**